mudlib/deprecated/alists_sorted.c

```diff
@@ -156,6 +157,11 @@ insert_alist(mixed key, varargs mixed * args)
         }
     }
 
+    /*
+    // update arg if it was given by reference
+    if (referencep(&(args[<1])))
+        args[<1] = alist;
+     */
```

Comment:
Is there a reason this is commented out?

Reply:
As far as I remember, the original efun never altered the argument, so this would silently introduce new functionality. But that's probably fine in this case, as the reference has to be passed in the call explicitly anyways.